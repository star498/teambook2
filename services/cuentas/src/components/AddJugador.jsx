import React from 'react';
 
const AddJugador = (props) => {
  return (
          
              <form onSubmit={(event) => props.addJugador(event)}>
              <h3><strong>Crear una cuenta</strong></h3>
                
                  <div class="form-group">
                  <input id="nombre" type="text" class="form-control" name="nombre"      value={props.nombre}  onChange={props.handleChange}  placeholder="Nombre " required/>
                  </div>
                 <div class="form-group">
                  <input id="apellido" type="text" class="form-control" name="apellido"     value={props.apellido}  onChange={props.handleChange}  placeholder="Apellido" required/>
                 </div>
                 <div class="form-group">
                  <input id="email" type="email" class="form-control" name="email" placeholder="Correo electronico"    onChange={props.handleChange}   value={props.email}  required/>
                 </div>
                 <div class="form-group">
                  <div class="input-group">
                   <span class="input-group-addon"><i class="glyphicon glyphicon-phone"></i></span>
                  <input id="celular" type="text" class="form-control" onChange={props.handleChange}  name="celular"      value={props.celular} placeholder="Numero" maxlength="9" required/>
                 </div>
                 </div>

                 <div class="form-group">
                    <div class="input-group">
                   <span class="input-group-addon"><i class="glyphicon glyphicon-calendar"></i></span>
                  <input id="fechanacimiento" type="Date" class="form-control" name="fechanacimiento" onChange={props.handleChange}   value={props.fechanacimiento} placeholder="Nacimiento" required/>
                 </div>
                 </div>
                <div class="form-group">
                  <div class="input-group">
                  <span class="input-group-addon"><i class="glyphicon glyphicon-user"></i></span>
                  <input id="usuario" type="text" class="form-control" name="usuario" placeholder="Usuario" required onChange={props.handleChange}   value={props.usuario} />
                </div>
                </div>
                
                <div class="form-group">
                   <div class="input-group">
                  <span class="input-group-addon"><i class="glyphicon glyphicon-lock"></i></span>
                  <input id="clave" type="password" class="form-control" name="clave" placeholder="ContraseÃ±a" required onChange={props.handleChange}   value={props.clave} />
                </div>
                </div>
               
                <div class="form-group">
                 <input  type="submit" value="registrar" name="submit" class="btn btn-success"/> 
                </div>
              </form>
            

  )
};
export default AddJugador;