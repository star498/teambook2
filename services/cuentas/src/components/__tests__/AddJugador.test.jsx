import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
 
import AddJugador from '../AddJugador';
 
test('AddJugador renders properly', () => {
  const wrapper = shallow(<AddJugador/>);
  const element = wrapper.find('form');
  expect(element.find('input').length).toBe(8);
  expect(element.find('input').get(0).props.name).toBe('nombre');
  expect(element.find('input').get(1).props.name).toBe('apellido');
  expect(element.find('input').get(2).props.name).toBe('email');
  expect(element.find('input').get(3).props.name).toBe('celular');
  expect(element.find('input').get(4).props.name).toBe('fechanacimiento');
  expect(element.find('input').get(5).props.name).toBe('usuario');
  expect(element.find('input').get(6).props.name).toBe('clave');
  expect(element.find('input').get(7).props.type).toBe('submit');
});


test('AddJugador renders a snapshot properly', () => {
  const tree = renderer.create(<AddJugador/>).toJSON();
  expect(tree).toMatchSnapshot();
});