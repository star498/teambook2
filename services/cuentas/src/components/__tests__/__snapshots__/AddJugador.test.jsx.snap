// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddJugador renders a snapshot properly 1`] = `
<form
  onSubmit={[Function]}
>
  <h3>
    <strong>
      Crear una cuenta
    </strong>
  </h3>
  <div
    className="field"
  >
    <input
      className="input is-large"
      name="nombre"
      placeholder="Ingrese nombre"
      required={true}
      type="text"
    />
  </div>
  <div
    className="field"
  >
    <input
      className="input is-large"
      name="apellido"
      placeholder="Ingrese apellido"
      required={true}
      type="text"
    />
  </div>
  <div
    className="field"
  >
    <input
      className="input is-large"
      name="email"
      placeholder="Ingrese contraseña"
      required={true}
      type="email"
    />
  </div>
  <div
    className="field"
  >
    <input
      className="input is-large"
      name="celular"
      placeholder="Celular"
      required={true}
      type="text"
    />
  </div>
  <div
    className="field"
  >
    <input
      className="input is-large"
      name="fechanacimiento"
      required={true}
      type="Date"
    />
  </div>
  <div
    className="field"
  >
    <input
      className="input is-large"
      name="usuario"
      placeholder="Usuario"
      required={true}
      type="text"
    />
  </div>
  <div
    className="field"
  >
    <input
      className="input is-large"
      name="clave"
      placeholder="Contraseña"
      required={true}
      type="text"
    />
  </div>
  <input
    className="button is-primary is-large is-fullwidth"
    type="submit"
    value="Submit"
  />
</form>
`;
